tests/ha/test_control_pod_restart.py

```diff
@@ -383,3 +401,286 @@ def test_failover_control_pod_kubectl(self):
 
         LOGGER.info("ENDED: Verify IOs before and after control pod fails over, verify control "
                     "pod failover. (using kubectl command)")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40370")
+    @CTFailOn(error_handler)
+    def test_rd_wr_del_during_ctrl_pod_failover(self):
+        """
+        Verify READs, WRITEs and DELETEs during control pod failover.
+        """
+        LOGGER.info("STARTED: Verify READs, WRITEs and DELETEs during control pod failover.")
+
+        event = threading.Event()  # Event to be used to send when data pods going down
+        wr_bucket = HA_CFG["s3_bucket_data"]["no_bck_background_deletes"]
+        LOGGER.info("Step 1: Perform WRITEs with variable object sizes on %s buckets "
+                    "for parallel DELETEs.", wr_bucket)
+        wr_output = Queue()
+        del_output = Queue()
+        remaining_bkt = 10
+        del_bucket = wr_bucket - remaining_bkt
+        users = self.mgnt_ops.create_account_users(nusers=2)
```

Comment:
why 2 users ?

Reply:
With single user delete was creating problem.

Reply:
> With single user delete was creating problem.

are we doing it for all the cases then ? what problem, is it due to control pod failure ?

Reply:
It deletes buckets created for read process, also created in write process.
Not needed for all cases, need different user for delete case only.